.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
I believe that this, combined with the absense of any issue templates in this repository, will prevent any issues from being created here, instead directing users over to https://github.com/FoldingAtHome/fah-issues.